liquibase-service/src/main/resources/db/rdb_modern/routines/018-sp_d_lab_test_postprocessing-001.sql

```diff
@@ -1374,8 +2320,7 @@ BEGIN
         FROM #LAB_TEST_final tlt
         EXCEPT
         SELECT lt.LAB_TEST_UID
-        FROM dbo.LAB_TEST lt;
-
+        FROM dbo.L_LAB_TEST lt;
```

Comment:
👍 
In the original sproc, was L_LAB_TEST used or LAB_TEST?

Reply:
LAB_TEST was used originally, changing it to L_LAB_TEST is what fixed the key generation issue